src/engine/source/builder/builders/stageBuilderNormalize.cpp

```diff
@@ -68,7 +68,30 @@ Builder getStageNormalizeBuilder(std::weak_ptr<Registry<Builder>> weakRegistry)
                                [registry, definitions](auto& tuple)
                                {
                                    auto& [key, value] = tuple;
-                                   if (allowedBlocks.count(key) == 0)
+                                   json::Json stageParseValue;
+                                   stageParseValue.setArray();
+                                   auto pos = key.find("parse|");
```

Comment:
Maybe the code could be improved by using a constant and calculating the size with the length function? In case the string changes tomorrow, it is not necessary to change anything else.

Example:
```
 const std::string PARSE = "parse|";
 self pos = key.find(PARSE);
 if (pos != std::string::npos)
 {
     auto field = key.substr(pos + PARSE.length());
```